lesson2/main.js

```diff
@@ -0,0 +1,20 @@
+const lists = document.getElementById('js-list');
+const list = document.createElement('li');
+const text = document.createTextNode('これです');
+
+const anchor = document.createElement('a');
+anchor.href = '1.html';
+
+const image = document.createElement('img');
+const src = document.createAttribute('src');
+const alt = document.createAttribute('alt');
+src.value = 'bookmark.png';
+alt.value = 'ブックマーク';
+image.setAttributeNode(src);
+image.setAttributeNode(alt);
+
+lists.appendChild(list);
+list.appendChild(anchor);
+anchor.appendChild(image);
+anchor.appendChild(text);
```

Comment:
This can also be specified directly in the anchor.
Therefore, "const text" is no longer needed.
IMO
```suggestion
anchor.textContent = "これです";
```

Reply:
I have a question.
Here is the result of the implementation.
[this](https://codesandbox.io/s/eager-dhawan-0wioy)

In textContent, the text is added just below the a tag.
I thought this was different from the implementation of this assignment.

What do you think...?

Reply:
I'm sorry. I thought it would be wrong to write all the answers, so I showed only a part of them. I apologize for any confusion I may have caused.
[IMO](https://codesandbox.io/s/jsleeson2-test-forked-5vk89?file=/main.js)

Reply:
I should have thought more carefully.
I apologize for taking so much of your time.

fixed.
The code is now shorter and easier to read.
I also learned some new methods.
Thank you.